### PR TITLE
[dockers] Default container autorestart feature to "enabled" for all except database

### PR DESCRIPTION
**Pull request by yozhao101**
Signed-off-by: Yong Zhao <yozhao@microsoft.com>

**- Why I did it**
Initially, the default state of autorestart feature is `disabled` for all containers. We change them to `enabled` except 
the database container such that we do not need
run extra `config` commands once this feature goes into production. If users do not want to adopt this feature,
they can disable it by running the `sudo config container feature autorestart <container_name> disabled`.

**- How I did it**

**- How to verify it**

### CODE REVIEW

File: files/build_templates/init_cfg.json.j2 [hunk 1]
```diff
@@ -28,7 +28,11 @@
 {%- for container in ["bgp", "database", "dhcp_relay", "lldp", "nat", "pmon", "radv", "restapi", "sflow",
                       "snmp", "swss", "syncd", "teamd", "telemetry"] %}
         "{{container}}": {
-            "auto_restart": "enabled",
+            {% if container == "database" %}
+                "auto_restart": "disabled",
+            {% else %}
+                "auto_restart": "enabled",
+            {% endif -%}
```
Comment (jleveque) [inline files/build_templates/init_cfg.json.j2:35]: The `"auto_restart": ...` lines are now indented too far. Also not sure about indenting the conditional lines, may add unnecessary whitespace. Have you tested this?

Comment (yozhao101) [reply files/build_templates/init_cfg.json.j2:35]: Good catch, It will introduce unnecessary whitespace.

Comment (jleveque) [reply files/build_templates/init_cfg.json.j2]: What about the indentation of the conditional lines? Again, have you tested it?

Comment (yozhao101) [reply files/build_templates/init_cfg.json.j2]: {
    "DEVICE_METADATA": {                                                                                                                                                
        "localhost": {
            "default_bgp_status": "up", 
            "default_pfcwd_status": "disable"
        }   
    },  
    "CRM": {
        "Config": {
            "polling_interval": "300",
            "ipv4_route_threshold_type": "percentage",
            "ipv4_route_low_threshold": "70",
            "ipv4_route_high_threshold": "85",
            "ipv6_route_threshold_type": "percentage",
            "ipv6_route_low_threshold": "70",
            "ipv6_route_high_threshold": "85",
            "ipv4_nexthop_threshold_type": "percentage",
            "ipv4_nexthop_low_threshold": "70",
            "ipv4_nexthop_high_threshold": "85",
            "ipv6_nexthop_threshold_type": "percentage",
            "ipv6_nexthop_low_threshold": "70",
            "ipv6_nexthop_high_threshold": "85",
            "ipv4_neighbor_threshold_type": "percentage",
            "ipv4_neighbor_low_threshold": "70",
            "ipv4_neighbor_high_threshold": "85",
            "ipv6_neighbor_threshold_type": "percentage",
            "ipv6_neighbor_low_threshold": "70",
            "ipv6_neighbor_high_threshold": "85",
            "nexthop_group_member_threshold_type": "percentage",
            "nexthop_group_member_low_threshold": "70",
            "nexthop_group_member_high_threshold": "85",
            "nexthop_group_threshold_type": "percentage",
            "nexthop_group_low_threshold": "70",
            "nexthop_group_high_threshold": "85",
            "acl_table_threshold_type": "percentage",
            "acl_table_low_threshold": "70",
            "acl_table_high_threshold": "85",
            "acl_group_threshold_type": "percentage",
            "acl_group_low_threshold": "70",
            "acl_group_high_threshold": "85",
            "acl_entry_threshold_type": "percentage",
            "acl_entry_low_threshold": "70",
            "acl_entry_high_threshold": "85",
            "acl_counter_threshold_type": "percentage",
            "acl_counter_low_threshold": "70",
            "acl_counter_high_threshold": "85",
            "fdb_entry_threshold_type": "percentage",
            "fdb_entry_low_threshold": "70",
            "fdb_entry_high_threshold": "85"
        }   
    },  
    "FEATURE": {
        "sflow": {
            "status": "disabled"
        },
        "telemetry": {
            "status": "enabled"
        }
    },  
    "CONTAINER_FEATURE": {
        "bgp": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "database": {
            
            "auto_restart": "disabled",
            "high_mem_alert": "disabled"
        },
        "dhcp_relay": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "lldp": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "nat": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "pmon": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "radv": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "restapi": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "sflow": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "snmp": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "swss": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "syncd": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "teamd": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
        "telemetry": {
            
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        }
    }   
}

Comment (yozhao101) [reply files/build_templates/init_cfg.json.j2]: When I tested, I missed the blank line,

Comment (jleveque) [reply files/build_templates/init_cfg.json.j2]: Why not just change the one line to:
```
        "auto_restart": "{% if container == "database" %}disabled{% else %}enabled{% endif %}",
```

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest mellanox please
